modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradePortletDisplayCheck.java

```diff
@@ -0,0 +1,181 @@
+			return _formatJSP(content);
+		}
+
+		JavaClass javaClass = JavaClassParser.parseJavaClass(fileName, content);
+
+		for (JavaTerm childJavaTerm : javaClass.getChildJavaTerms()) {
+			if (!childJavaTerm.isJavaMethod()) {
+				continue;
+			}
+
+			JavaMethod javaMethod = (JavaMethod)childJavaTerm;
+
+			String javaMethodContent = javaMethod.getContent();
+
+			Matcher getPortletInstanceConfigurationMatcher =
+				_getPortletInstanceConfigurationPattern.matcher(
+					javaMethodContent);
+
+			while (getPortletInstanceConfigurationMatcher.find()) {
+				String methodCall = JavaSourceUtil.getMethodCall(
+					javaMethodContent,
+					getPortletInstanceConfigurationMatcher.start());
+
+				if (!hasClassOrVariableName(
+						"PortletDisplay", content, fileName, methodCall)) {
+
+					continue;
+				}
+
+				List<String> parameterList = JavaSourceUtil.getParameterList(
+					methodCall);
+
+				String indent = JavaSourceUtil.getIndent(methodCall);
+
+				StringBundler sb = new StringBundler(3);
+
+				sb.append(indent);
+				sb.append("ConfigurationProviderUtil");
+				sb.append(getPortletInstanceConfigurationMatcher.group(2));
+
+				String newJavaMethod = null;
+
+				Matcher themeDisplayMatcher = _themeDisplayPattern.matcher(
+					javaMethodContent);
+
+				if (themeDisplayMatcher.find()) {
+					newJavaMethod = StringUtil.replace(
+						javaMethodContent, methodCall,
+						JavaSourceUtil.addMethodNewParameters(
+							indent, new int[] {parameterList.size()},
+							sb.toString(),
+							new String[] {themeDisplayMatcher.group(1)},
+							parameterList));
+				}
+				else {
+					String themeDisplayCall =
+						getPortletInstanceConfigurationMatcher.group(1) +
+							".getThemeDisplay()";
+
+					newJavaMethod = StringUtil.replace(
+						javaMethodContent, methodCall,
+						JavaSourceUtil.addMethodNewParameters(
+							indent, new int[] {parameterList.size()},
+							sb.toString(), new String[] {themeDisplayCall},
+							parameterList));
+				}
+
+				newContent = StringUtil.replace(
+					newContent, javaMethodContent, newJavaMethod);
+			}
+		}
+
+		return newContent;
+	}
+
+	@Override
+	protected String[] getNewImports() {
+		return new String[] {
+			"com.liferay.portal.configuration.module.configuration." +
+				"ConfigurationProviderUtil"
+		};
+	}
+
+	@Override
+	protected String[] getValidExtensions() {
+		return new String[] {"java", "jsp"};
+	}
+
+	private String _formatJSP(String content) {
+		String newContent = content;
+
+		Matcher matcher = _getPortletInstanceConfigurationPattern.matcher(
+			content);
+
+		while (matcher.find()) {
+			String methodCall = JavaSourceUtil.getMethodCall(
+				content, matcher.start());
+
+			String variableName = getVariableName(methodCall);
+
+			if (!variableName.contains("portletDisplay")) {
+				continue;
+			}
+
+			List<String> parameterList = JavaSourceUtil.getParameterList(
+				methodCall);
+
+			String indent = JavaSourceUtil.getIndent(methodCall);
```

Comment:
@TamyrisTorres, please declare the variables following the order they're being used, e.g. the first declared variable is the first to be used, the second declared variable is the second to be used, etc

Reply:
resolved